migrations/20260610135812_add_cancelling_cancelled_to_offchain_order_view.sql

```
@@ -0,0 +1,27 @@
+            WHEN json_extract(payload, '$.Live.Submitted') IS NOT NULL THEN 'Submitted'
+            WHEN json_extract(payload, '$.Live.PartiallyFilled') IS NOT NULL THEN 'PartiallyFilled'
+            WHEN json_extract(payload, '$.Live.Cancelling') IS NOT NULL THEN 'Cancelling'
+            WHEN json_extract(payload, '$.Live.Filled') IS NOT NULL THEN 'Filled'
+            WHEN json_extract(payload, '$.Live.Failed') IS NOT NULL THEN 'Failed'
+            WHEN json_extract(payload, '$.Live.Cancelled') IS NOT NULL THEN 'Cancelled'
+        END
+    ) STORED
+);
+
+CREATE INDEX IF NOT EXISTS idx_offchain_order_view_status
+    ON offchain_order_view(status) WHERE status IS NOT NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)